src/components/TextInput/TextInput.module.scss

```diff
@@ -46,3 +46,25 @@
   background-color: var(--GrayStrokeAndDisabled--translucent);
   border-color: transparent;
 }
+
+.TextInputWrapper{
```

Comment:
add space between `TextInputWrapper` and `{`

Reply:
added. Thank you